packages/meditrak-server/src/apiV2/postChanges.js

```diff
@@ -182,8 +182,10 @@ const requiresSurveyResponseTranslation = (
   const answerTranslatorFields = Object.keys(answerTranslators);
```

Comment:
This file applies a server-side translation, so that even people with the existing version of the app can submit data using it, and it'll strip out the empty string answers at this point